list_to_email_converter.py

```diff
@@ -0,0 +1,178 @@
+            akt_lines = akt_lines+1
+            if (line ==''):
+                email.close()
+                email = open('_data/people.yml', 'r')
+            if line.find(people[i]+":")!= -1:
+                helpvar = 1
+            elif helpvar > 0 and line.find('  sur_name:') != -1:
+                name = name + line.replace("  sur_name: ", "").rstrip()
+                helpvar = helpvar + 1
+            elif helpvar > 0 and line.find('  given_name:') != -1:
+                name = line.replace("  given_name: ", "").rstrip() + " " + name
+                helpvar = helpvar + 1
+            elif helpvar > 0 and line.find('  email:') != -1:
+                emailaddress = line.replace("  email: ", "").rstrip()
+                helpvar = helpvar + 1
+            elif helpvar == 4:
+                break
+            if (akt_lines >= num_lines):
+                email.seek(0)
+                akt_lines = 0
+
+        head.append([people[i], name, emailaddress])
+        realname[people[i]] = name
+        i=i+1
+        ## checks email and real name of the projecleaders
+
+    else:
+        while people[i] == people[i + 1]:
+            people.pop(i+1)
+        if people[i+1].find('*') != -1 and people[i] == people[i+1].replace("*", ""):
+            i=i+1
+            continue
+            ## if one person is projectleader
+        helpvar = 0
+        name = ""
+        for line in email:
+            akt_lines = akt_lines + 1
+            if (line == ''):
+                email.close()
+                email = open('_data/people.yml', 'r')
+            if line.find(people[i]) != -1:
+                helpvar = 1
+            elif helpvar > 0 and line.find('  sur_name:') != -1:
+                name = name + line.replace("  sur_name:", "").rstrip()
+                helpvar = helpvar + 1
+            elif helpvar > 0 and line.find('  given_name:') != -1:
+                name = line.replace("  given_name: ", "").rstrip() + " " + name
+                helpvar = helpvar + 1
+            elif helpvar == 3:
+                break
+            if (akt_lines >= num_lines):
+                email.seek(0)
+                akt_lines = 0
+        realname[people[i]]= name
+        i=i+1
+        ## checks the real names of the persons
+
+
+
+print (people)
+print (head)
+print (realname)
+
+email.close()
+
+datei = open("projectleader_mailinglist.txt", "w")
+
+for element in head:
+    datei.write(element[2] +", ")
+
+datei.close()
+
+
+file = open('mailinglist_pms.txt', 'w')
+file.write("Head;Link\n")
+
+for key, value in iter(link_pair.items()):
+    link = "https://docs.google.com/forms/d/1HtvPVp-oCxFACzIgJD49g5qXjt8otU9YIalt-GOCIAg/viewform?entry.1218238180=[projectname]&entry.703889404=[pms]"
+
+    link = link.replace("[projectname]", key)
+    endung = quote(": 0.0 PMs")
+    persons = ""
+    for item in value:
+        if (item.find('*') != -1):
+            item = item.replace("*", "")
+            file.write(realname.get(item) + " ; ")
```

Comment:
I need the email address here, not the real name (sorry, if you or I got confused this morning)..